### ISSUE TITLE
color coding of model & corpus in header

### ISSUE THREAD

**Issue by colinallen** [inpho/inphosemantics-frontend#25]
I think this is a simple request and I'd like to make it reasonably high priority...

Currently it's very hard to find the columns one wants to compare in a display such as this one: 

http://inphodata.cogs.indiana.edu:8080/?corpora=sep.complete%2Ciep.complete&models=beagle.composite%2Cbeagle.context%2Cbeagle.order&phrases=kant%2Cdescartes&searchLimit=24

What would help is to assign a fixed color to each corpus and to each model, and then to use that color in the background of the corpus and model terms in the header of each panel.

Here are suggested color schemes:

SEP: red
IEP: light blue
PhilPapers: green
Malaria: yellow

Beagle-Environment: grey
Beagle-Context: blue
Beagle-Order: yellow
Beagle-Composite: light green
tf: pink
tf-idf: purple

**Comment by srwaggon** [comment]
This isn't yet live, but let me know what you think and I'll see about getting it live soon.
http://inphodata.cogs.indiana.edu:9090/?corpora=sep.complete&models=beagle.environment%2Cbeagle.context&phrases=kant&searchLimit=20

**Comment by colinallen** [comment]
Great - yes, that's basically exactly what I had in mind although I think it's ok to go to the abbreviated labels (sep.complete, beagle.environment, etc.) to avoid the wide columns and wasted space.  Thanks!

**Comment by srwaggon** [comment]
Great, so this is an interesting and actually very awesome feature which I love, but it makes me want to extend it toward something similar for the phrases.

I intend to later open a separate bug report for some sort of choose-n-colours algorithm for phrases.

**Comment by colinallen** [comment]
Indeed, yes, it would be great to extend this to the terms themselves, but good to add as separate feature-request ticket, as you say

**Comment by colinallen** [comment]
Please don't close issue until released to master branch.  Thanks!
